File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/coinmarketcap/Quotes.java
package com.androidevlinux.percy.UTXO.data.models.coinmarketcap;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Quotes {

    @SerializedName("USD")
    @Expose
    private USD uSD;

    public USD getUSD() {
        return uSD;
    }

    public void setUSD(USD uSD) {
        this.uSD = uSD;
    }

}

File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/coinmarketcap/Quotes.kt
package com.androidevlinux.percy.UTXO.data.models.coinmarketcap

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Quotes {

    @SerializedName("USD")
    @Expose
    var usd: USD? = null

}